docs/user-manuals/imagepulljob.md

```diff
@@ -13,6 +13,14 @@ Users can create an ImagePullJob to declare an image should be downloaded on whi
 Note that the NodeImage is quite **a low-level API**. You should only use it when you prepare to pull an image on a definite Node.
 Otherwise, you should **use the ImagePullJob to pull an image on a batch of Nodes.**
 
+
+## Feature-gate
+**Since kruise v1.5.0** ImagePullJob/ImageListPullJob feature is turned off by default due to permissions, if you want to turn it on set feature-gate *ImagePullJobGate*.
```

Comment:
Since kruise v1.5.0** ImagePullJob/ImageListPullJob feature is turned off by default to reduce the privilege of default installation. One ca turn it on by setting feature-gate *ImagePullJobGate*.